_sass/_page.scss

```
@@ -1,21 +1,17 @@
----
----
-
 /*
  * Reset
  */
 
 * {
   -webkit-box-sizing: border-box;
-     -moz-box-sizing: border-box;
-          box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
 }
 
 html {
   font: 14px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Bump this to use 16px and GitHub's font stack.

```suggestion
  font: 16px/1.5 -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji;
```